szamlazz/models.py

```diff
@@ -147,8 +147,8 @@ def __init__(self,
         self.xml_namespace = xml_namespace
         self.__response = response
         self.__action_success: bool = False
-        content_type = response.headers.get("Content-Type")
-        if content_type == "application/octet-stream":
+        content_type = response.headers.get("Content-Type", "")
```

Comment:
According to [RFC2616, section 7.2.1](https://www.rfc-editor.org/rfc/rfc2616#section-7.2.1), the `Content-Type` header should be present in the response. Setting a default value of `""` is a wrong approach in my opinion because that would cover an implementation error on Szamlazz.hu's end.

If Szamlazz.hu is returning non-RFC compliant responses, then this client should actually fail but I'm open to discuss this matter further.